src/main/resources/reference.conf

```diff
@@ -123,6 +123,69 @@ updatealerts=true
 lyrics.default = "A-Z Lyrics"
 
 
+// These settings allow you to configure custom aliases for all commands.
+// Multiple aliases may be given, separated by commas.
+//
+// Example 1: Giving command "play" the alias "p":
+// play = [ p ]
+//
+// Example 2: Giving command "search" the aliases "yts" and "find":
+// search = [ yts, find ]
+
+aliases {
+
+  // General commands
+
+  settings = [ status ]
+
+
+  // Music commands
+
+  lyrics = []
+  nowplaying = [ np, current ]
+  play = []
+  playlists = [ pls ]
+  queue = [ list ]
+  remove = [ delete ]
+  scsearch = []
+  search = [ ytsearch ]
+  shuffle = []
+  skip = [ voteskip ]
+
+
+  // Admin commands
+
+  setdj = []
+  settc = []
+  setvc = []
+
+
+  // DJ Commands
+
+  forceremove = [ forcedelete, modremove, moddelete ]
+  forceskip = [ modskip ]
+  movetrack = [ move ]
+  pause = []
+  playnext = []
+  repeat = []
+  skipto = [ jumpto ]
+  stop = []
+  volume = [ vol ]
+
+
+  // Owner commands
+
+  autoplaylist = []
+  debug = []
+  eval = []
+  playlist = []
+  setavatar = []
+  setgame = []
+  setname = []
+  setstatus = []
+  shutdown = []
+}
+
```

Comment:
I'm still not sure I want to list every command here; something being not included should just default to an empty alias list. That being said, this is fine for now; I might just edit the example config or adjust this later.